server/models/subscription.ts

```diff
@@ -31,6 +31,12 @@ export interface SubscriptionAttributes {
   User?: UserAttributes;
   NotificationCategory?: NotificationCategoryAttributes;
   Notifications?: NotificationAttributes[];
+  chain_id?: string;
+  community_id?: string;
+  offchain_thread_id?: number;
+  offchain_comment_id?: number;
+  chain_event_type_id?: string;
+  chain_entity_id?: string;
 }
```

Comment:
Depending how we want to invoke these associations (i.e. if we use `include` in queries), you may want to also add optional properties of `Chain`, `OffchainCommunity`, `OffchainThread`, etc. that refer to the Attributes object of each, rather than only adding the raw identifiers.

Reply:
👍